hepcrawl/spiders/aps_spider.py

```diff
@@ -55,9 +54,10 @@ class APSSpider(LastRunStoreSpider):
 
     @strict_kwargs
     def __init__(self, from_date=None, until_date=None,
-                 date="published", sets=None, per_page=100,
+                 date="published", sets=None, per_page=100, aps_url=None,
                  **kwargs):
         """Construct APS spider."""
+        self.aps_url = aps_url or self.aps_base_url
```

Comment:
maybe remove `aps_base_url` and put it as default for the `aps_url` arg?

Reply:
:+1: